.gitignore

```diff
@@ -93,3 +93,9 @@ ENV/
 
 # hatch-vcs
 src/*/_version.py
+
+*.bak
+*.swp
+
+CLAUDE.local.md
+.claude/*
```

Comment:
No objection to adding these here, but you may find it easier to add them a personal, global `.gitignore`.